pages/createUpdate.js

```diff
@@ -89,7 +91,23 @@ class CreateUpdatePage extends React.Component {
       if (update.isChangelog) {
         update.isPrivate = false;
       }
-      const res = await this.props.createUpdate({ variables: { update } });
+      const res = await this.props.createUpdate({
+        variables: { update },
+        refetchQueries: [
+          {
+            query: updatesQuery,
+            context: API_V2_CONTEXT,
+            variables: {
+              collectiveSlug: this.props.slug,
+              offset: 0,
+              limit: UPDATES_PER_PAGE,
+              includeHostedCollectives: true,
+              orderBy: { field: 'CREATED_AT', direction: 'DESC' },
+            },
```

Comment:
Maybe we could simplify this and make sure we don't break it in the future by using `getUpdatesVariables` as a common helper:

```es6
getUpdatesVariables = (collectiveSlug, limit = UPDATES_PER_PAGE) => { ... }
```

Reply:
@Betree : I've re-used the `getUpdatesVariables` function from `UpdatesWithData.js` with some small modifications. I think that looks better, let me know.